# Translations template for glance.
# Copyright (C) 2016 ORGANIZATION
# This file is distributed under the same license as the glance project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2016.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: glance 12.0.1.dev43\n"
"Report-Msgid-Bugs-To: https://bugs.launchpad.net/openstack-i18n/\n"
"POT-Creation-Date: 2016-04-19 06:56+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.2.0\n"

#: glance/location.py:436 glance/location.py:459
#, python-format
msgid "Successfully verified signature for image %s"
msgstr ""

#: glance/scrubber.py:213
#, python-format
msgid "Starting Daemon: wakeup_time=%(wakeup_time)s threads=%(threads)s"
msgstr ""

#: glance/scrubber.py:228
msgid "Daemon Shutdown on KeyboardInterrupt"
msgstr ""

#: glance/scrubber.py:240
#, python-format
msgid "Initializing scrubber with configuration: %s"
msgstr ""

#: glance/scrubber.py:298
#, python-format
msgid "Scrubbing image %(id)s from %(count)d locations."
msgstr ""

#: glance/scrubber.py:312
#, python-format
msgid "Image %s has been scrubbed successfully"
msgstr ""

#: glance/scrubber.py:326
#, python-format
msgid ""
"Image location for image '%s' not found in backend; Marking image "
"location deleted in db."
msgstr ""

#: glance/scrubber.py:335
#, python-format
msgid "Image %s is scrubbed from a location."
msgstr ""

#: glance/api/middleware/cache.py:59
msgid "Initialized image cache middleware"
msgstr ""

#: glance/api/middleware/cache_manage.py:73
msgid "Initialized image cache management middleware"
msgstr ""

#: glance/api/middleware/gzip.py:35
msgid "Initialized gzip middleware"
msgstr ""

#: glance/api/v1/images.py:749
#, python-format
msgid "Uploaded data of image %s from request payload successfully."
msgstr ""

#: glance/api/v1/images.py:814
msgid "Triggering asynchronous copy from external source"
msgstr ""

#: glance/api/v1/upload_utils.py:123
#, python-format
msgid "Cleaning up %s after exceeding the quota"
msgstr ""

#: glance/api/v2/image_actions.py:49
#, python-format
msgid "Image %s is deactivated"
msgstr ""

#: glance/api/v2/image_actions.py:65
#, python-format
msgid "Image %s is reactivated"
msgstr ""

#: glance/api/v2/image_data.py:109
#, python-format
msgid "Unable to create trust: %s Use the existing user token."
msgstr ""

#: glance/api/v2/image_data.py:132
#, python-format
msgid "Unable to delete trust %(trust)s: %(msg)s"
msgstr ""

#: glance/async/flows/base_import.py:375
#, python-format
msgid "%(task_id)s of %(task_type)s completed"
msgstr ""

#: glance/cmd/replicator.py:371
#, python-format
msgid "Storing: %s"
msgstr ""

#: glance/cmd/replicator.py:448
#, python-format
msgid "Considering: %s"
msgstr ""

#: glance/cmd/replicator.py:474 glance/cmd/replicator.py:548
#, python-format
msgid "Image %s metadata has changed"
msgstr ""

#: glance/cmd/replicator.py:554
#, python-format
msgid "Image %s is being synced"
msgstr ""

#: glance/common/wsgi.py:331
#, python-format
msgid "Starting %d workers"
msgstr ""

#: glance/common/wsgi.py:344
#, python-format
msgid "Removed dead child %s"
msgstr ""

#: glance/common/wsgi.py:347
#, python-format
msgid "Removed stale child %s"
msgstr ""

#: glance/common/wsgi.py:359
msgid "All workers have terminated. Exiting"
msgstr ""

#: glance/common/wsgi.py:376
msgid "Caught keyboard interrupt. Exiting."
msgstr ""

#: glance/common/wsgi.py:457
#, python-format
msgid "Child %d exiting normally"
msgstr ""

#: glance/common/wsgi.py:462
#, python-format
msgid "Started child %s"
msgstr ""

#: glance/common/wsgi.py:492
msgid "Starting single process server"
msgstr ""

#: glance/common/glare/loader.py:126 glance/common/glare/loader.py:150
#, python-format
msgid "Artifact %s has been successfully loaded"
msgstr ""

#: glance/common/scripts/__init__.py:30
#, python-format
msgid "Loading known task scripts for task_id %(task_id)s of type %(task_type)s"
msgstr ""

#: glance/common/scripts/image_import/main.py:36
#, python-format
msgid "Task %(task_id)s beginning import execution."
msgstr ""

#: glance/common/scripts/image_import/main.py:147
#, python-format
msgid "Task %(task_id)s: Got image data uri %(data_uri)s to be imported"
msgstr ""

#: glance/common/scripts/image_import/main.py:156
#, python-format
msgid "Task %(task_id)s: Could not import image file %(image_data)s"
msgstr ""

#: glance/db/simple/api.py:59
#, python-format
msgid "Calling %(funcname)s: args=%(args)s, kwargs=%(kwargs)s"
msgstr ""

#: glance/db/simple/api.py:65
#, python-format
msgid "Returning %(funcname)s: %(output)s"
msgstr ""

#: glance/db/simple/api.py:2003
#, python-format
msgid "Could not find artifact %s"
msgstr ""

#: glance/db/simple/api.py:2007
msgid "Unable to get deleted image"
msgstr ""

#: glance/db/sqlalchemy/api.py:1299
#, python-format
msgid "Purging deleted rows older than %(age_in_days)d day(s) from table %(tbl)s"
msgstr ""

#: glance/db/sqlalchemy/api.py:1316
#, python-format
msgid "Deleted %(rows)d row(s) from table %(tbl)s"
msgstr ""

#: glance/db/sqlalchemy/metadata.py:161
#, python-format
msgid "Table %s has been cleared"
msgstr ""

#: glance/db/sqlalchemy/metadata.py:235
#, python-format
msgid "Overwriting namespace %s"
msgstr ""

#: glance/db/sqlalchemy/metadata.py:251
#, python-format
msgid "Skipping namespace %s. It already exists in the database."
msgstr ""

#: glance/db/sqlalchemy/metadata.py:342
#, python-format
msgid "File %s loaded to database."
msgstr ""

#: glance/db/sqlalchemy/metadata.py:344
msgid "Metadata loading finished"
msgstr ""

#: glance/db/sqlalchemy/metadata.py:452
#, python-format
msgid "Namespace %(namespace)s saved in %(file)s"
msgstr ""

#: glance/db/sqlalchemy/migrate_repo/schema.py:100
#, python-format
msgid "creating table %(table)s"
msgstr ""

#: glance/db/sqlalchemy/migrate_repo/schema.py:106
#, python-format
msgid "dropping table %(table)s"
msgstr ""

#: glance/db/sqlalchemy/migrate_repo/versions/017_quote_encrypted_swift_credentials.py:66
msgid ""
"'metadata_encryption_key' was not specified in the config file or a "
"config file was not specified. This means that this migration is a NOOP."
msgstr ""

#: glance/domain/__init__.py:400
#, python-format
msgid "Task [%(task_id)s] status changing from %(cur_status)s to %(new_status)s"
msgstr ""

#: glance/image_cache/__init__.py:69
#, python-format
msgid "Image cache loaded driver '%s'."
msgstr ""

#: glance/image_cache/__init__.py:78 glance/image_cache/__init__.py:97
msgid "Defaulting to SQLite driver."
msgstr ""

#: glance/image_cache/prefetcher.py:83
#, python-format
msgid "Successfully cached all %d images"
msgstr ""

#: glance/image_cache/drivers/sqlite.py:409
#: glance/image_cache/drivers/xattr.py:338
#, python-format
msgid "Not queueing image '%s'. Already cached."
msgstr ""

#: glance/image_cache/drivers/sqlite.py:413
#: glance/image_cache/drivers/xattr.py:342
#, python-format
msgid "Not queueing image '%s'. Already being written to cache"
msgstr ""

#: glance/image_cache/drivers/sqlite.py:418
#: glance/image_cache/drivers/xattr.py:347
#, python-format
msgid "Not queueing image '%s'. Already queued."
msgstr ""

#: glance/image_cache/drivers/sqlite.py:435
#, python-format
msgid "Removed invalid cache file %s"
msgstr ""

#: glance/image_cache/drivers/sqlite.py:449
#, python-format
msgid "Removed stalled cache file %s"
msgstr ""

#: glance/image_cache/drivers/xattr.py:392
#, python-format
msgid "Reaped %(reaped)s %(entry_type)s cache entries"
msgstr ""

#: glance/quota/__init__.py:332
#, python-format
msgid "Cleaning up %s after exceeding the quota."
msgstr ""

#: glance/registry/api/v1/images.py:364 glance/registry/api/v1/images.py:403
#: glance/registry/api/v1/images.py:507
#, python-format
msgid "Image %(id)s not found"
msgstr ""

#: glance/registry/api/v1/images.py:369 glance/registry/api/v1/images.py:399
#: glance/registry/api/v1/images.py:517
#, python-format
msgid "Access denied to image %(id)s but returning 'not found'"
msgstr ""

#: glance/registry/api/v1/images.py:391
#, python-format
msgid "Successfully deleted image %(id)s"
msgstr ""

#: glance/registry/api/v1/images.py:394
#, python-format
msgid "Delete denied for public image %(id)s"
msgstr ""

#: glance/registry/api/v1/images.py:431
#, python-format
msgid "Rejecting image creation request for invalid image id '%(bad_id)s'"
msgstr ""

#: glance/registry/api/v1/images.py:443
#, python-format
msgid "Successfully created image %(id)s"
msgstr ""

#: glance/registry/api/v1/images.py:499
#, python-format
msgid "Updating metadata for image %(id)s"
msgstr ""

#: glance/registry/api/v1/images.py:512
#, python-format
msgid "Update denied for public image %(id)s"
msgstr ""

#: glance/registry/api/v1/members.py:198
#, python-format
msgid "Successfully updated memberships for image %(id)s"
msgstr ""

#: glance/registry/api/v1/members.py:271
#, python-format
msgid "Successfully updated a membership for image %(id)s"
msgstr ""

#: glance/registry/api/v1/members.py:318
#, python-format
msgid "Successfully deleted a membership from image %(id)s"
msgstr ""

